frontend/LockedIn/components/AddChallenge.js

```diff
@@ -35,77 +42,78 @@ const AddChallenge = ({ navigation }) => {
 
   const createAndUploadChallenge = async () => {
     if (!selectedImage) {
-      alert('Please select an image for the challenge.');
+      alert("Please select an image for the challenge.");
       return;
     }
 
-    const filename = selectedImage.split('/').pop();
+    const filename = selectedImage.split("/").pop();
     const match = /\.(\w+)$/.exec(filename);
     const type = match ? `image/${match[1]}` : `image`;
 
-    console.log('Selected Image URI:', selectedImage); // Debug log
-    console.log('Filename:', filename); // Debug log
-
     const formData = new FormData();
-    formData.append('title', challengeTitle);
-    formData.append('email', 'user@example.com'); // This should be the logged-in user's email
-    formData.append('image', {
+    formData.append("title", challengeTitle);
+    formData.append("username", "username"); // Replace 'username' with the logged-in user's username
+    formData.append("image", {
       uri: selectedImage,
       name: filename,
       type,
     });
 
     try {
-      const response = await api.post('/create_challenge', formData, {
+      const response = await api.post("/create_challenge", formData, {
         headers: {
-          'Content-Type': 'multipart/form-data',
+          "Content-Type": "multipart/form-data",
         },
       });
 
-      console.log('Server Response:', response.data); // Debug log
+      console.log("Server Response:", response.data);
 
       if (response.data.success) {
-        Alert.alert('Success', 'Challenge created with image!');
+        Alert.alert("Success", "Challenge created with image!");
         navigation.goBack();
       } else {
-        alert('Failed to create challenge.');
+        alert("Failed to create challenge.");
       }
     } catch (error) {
-      console.error('Error:', error); // Debug log
-      alert('Success!');
+      console.error("Error:", error);
+      alert("Failed to create challenge.");
     }
   };
+
   const sendImageToAI = async () => {
     if (!selectedImage) {
-      alert('Please select an image to send to the AI.');
+      alert("Please select an image to send to the AI.");
       return;
     }
 
-    const filename = selectedImage.split('/').pop();
+    const filename = selectedImage.split("/").pop();
     const match = /\.(\w+)$/.exec(filename);
     const type = match ? `image/${match[1]}` : `image`;
 
     const formData = new FormData();
-    formData.append('user_image', {
+    formData.append("user_image", {
       uri: selectedImage,
       name: filename,
       type,
     });
 
     try {
-      const response = await axios.post('https://ayush-login.saipriya.org/prompt_img', formData, {
-        headers: {
-          'Content-Type': 'multipart/form-data',
-        },
-      });
+      const response = await axios.post(
+        "https://ayush-login.saipriya.org/prompt_img",
```

Comment:
this should be a .env link instead of your server hardcoded into here

Reply:
removed the code entirely, dont need to launch with this feature